tasks/tor.yml

```diff
@@ -20,6 +20,16 @@
   register: descriptor_cookie
   when: ssh_onion and torrc.changed
 
+- name: Grab sandstorm onion address
+  shell: cat /var/lib/tor/sandstorm/hostname | awk '{print $1}'
+  register: sandstorm_onion_address
+  when: sandstorm_onion and torrc.changed
+
+- name: Grab sandstorm client descriptor cookie
```

Comment:
We don't currently configure HiddenServiceAuth for the Sandstorm HS - although we could add it as an option.

Right now though this will always be blank.